api/scpca_portal/test/management/commands/test_load_data.py

```diff
@@ -26,8 +32,34 @@ def setUp(self):
     @classmethod
     def tearDownClass(cls):
         super().tearDownClass()
+        if SAVE_README_OUTPUT:
+            cls.test_readme_files()
         shutil.rmtree(common.OUTPUT_DATA_PATH, ignore_errors=True)
 
+    @classmethod
+    def test_readme_files(cls):
+        if not SAVE_README_OUTPUT:
+            return
+        # Make sure to instantiate TestLoadData to use an instance method
+        self = cls()
+        # Make sure to create README_DIR if it doesn't exist to prevent an error
+        README_DIR.mkdir(parents=True, exist_ok=True)
+
+        for zip_path in common.OUTPUT_DATA_PATH.glob("*.zip"):
+            # Rename 'RERADME.md' with the name of the zip file
+            readme_name = f"{zip_path.stem}.md"
+            readme_output_path = README_DIR / readme_name
+            with ZipFile(zip_path, "r") as zip_file:
+                if README_FILE in zip_file.namelist():
+                    with zip_file.open(README_FILE) as readme_file:
+                        # Replace 3 or more lines with double newlines before saving
+                        formatted_content = re.sub(
+                            r"\n{3,}", "\n\n", readme_file.read().decode(ENCODING)
+                        ).strip()
+                        with readme_output_path.open("w", encoding=ENCODING) as output_file:
+                            output_file.write(formatted_content)
+                    self.assertProjectReadmeContent(readme_output_path, formatted_content)
```

Comment:
This is a great start but we don't actually want to have a test that generates test data this way. We just wanted to do this to boostrap the test data folder. We should update this test so that it explicitly checks specific readme's for complete coverage. That should amount to only 4 sample readmes, 1 metadata file only readme for now, and then the project combinations.

Reminder that we want the new flow for suggested changes to be, PR manual changes to edit copy to the test data then we will update the templates until they pass.

Reply:
So after looking at the updates I think the next step should be:
- remove the `test_readme_files` classmethod, we dont want a test that creates side effects
- i think we dont want to have one test that does all the readmes, instead lets remove `assertProjectReadmeContains` with `assertProjectReadmeContent` so we can validate it when we create the computed file
- Let's remove the addition of ENCODING, and remove the redundancy in another PR

Reply:
I just wanted to clarify that it is a good point that your suggestion for readme names exposes. We dont need the sample versions since they are identical to projects with the same configurations.

So your list of files is sufficiently complete. You just need to grab the correct file. I think prefixing them with the `scpca_id` is what we want to help specify which project id we expect to see in the file's contents. That seems a bit easier to maintain to me.

Reply:
I've made the following updates: 
- Regenerated the readme files in `test_data/readmes` (at 7942ce22000940a3321db3d911d35e34e74862c1)
- Removed the constants and the class method used for saving readme files outputs (at 6d26da78a17c066ee9df06a0c150789f2485999a)
- Refactored `assertProjectReadmeContent` and replaced `assertProjectReadmeContains` (deleted) with it (at 7f618e55383688fa20a2f0f1bc50fd8c7b8a4b52)

This PR is ready for another review. Thank you David!